app/tests/test_settings.py

```diff
@@ -6,6 +6,12 @@
 from app.settings import TwingateOperatorSettings
 
 
+@pytest.fixture()
+def mock_get_remote_network_by_name():
+    with patch("app.api.TwingateAPIClient.get_remote_network_by_name") as m:
```

Comment:
Maybe we want to use [`monkeypatch`](https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html) fixture make this even more pytest-y 😛

Reply:
I think it has diff purpose